src/dmd/backend/backconfig.d

```diff
@@ -50,6 +50,7 @@ extern (C) void out_config_init(
         bool alwaysframe,       // always create standard function frame
         bool stackstomp,        // add stack stomping code
         ubyte avx,              // use AVX instruction set (0, 1, 2)
+        ubyte pic,              // position independence level (0, 1, 2)
```

Comment:
Why is this not an enum?

Reply:
Because global.h is used only by gdc/ldc, and I believe they have their own methods of specifying PIC, i.e. it's simply a placeholder for them.

Reply:
Indeed, this field is ignored by gdc.

Reply:
So, if this is ignored by GDC, why do we need to adopt the code for them?

Reply:
I've just realised this line is of a backend module, which makes my contribution to this conversation only serving to further confuse.

I suspect the real reasoning here would be the division between front and back ends.  You can't import the enum here, so the base type is used instead.

Reply:
So add an enum in the backend as well. Or put it in a place where both the frontend and the backend can import it.